plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -528,6 +566,7 @@ private String createContainer(String workspaceId,
               .withEntrypoint(toArrayIfNotNull(service.getEntrypoint()))
               .withLabels(service.getLabels())
               .withNetworkingConfig(networkingConfig)
+              .withUser(cheUserId)
```

Comment:
Yes, it is. The ContainerConfig class uses the empty string. See https://github.com/eclipse/che/blob/master/plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/ContainerConfig.java#L44